src/containers/pages/InterventionPlan/UpdatePlan/index.tsx

```diff
@@ -43,13 +43,15 @@ const UpdatePlan = (props: RouteComponentProps<RouteParams> & UpdatePlanProps) =
   if (!planIdentifier) {
     return null; /** we should make this into a better error page */
   }
+  const apiService = new service(OPENSRP_PLANS);
```

Comment:
Also with the `apiService` defined before being called  was causing a problem which have not yet been able to understand fully.

Reply:
This probably came from the more aggressive linter that we are now using.